lib/featuring/persistence/adapter.rb

```diff
@@ -162,7 +164,7 @@ def reload
         def transaction
           transaction = Transaction.new(self)
           yield transaction
-          create_or_update_feature_flags(**transaction.values)
+          create_or_update_feature_flags(__perform: :replace, **transaction.values)
```

Comment:
Is it safe to set `__perform` here (and below)? I haven't dug into how stable that might be in AR, but the fact it's double-underscored gives me pause.

Reply:
Should be fine, since we own these methods. I prefixed the kwarg with a double underscore to prevent collisions with feature flag names (since those are also passed as kwargs).

Reply:
Ah, makes sense. Thanks.